.github/workflows/deploy.yml

```diff
@@ -17,11 +17,17 @@ jobs:
     - name: Build
       run: make web
 
+    - name: Upload web artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: www
+        path: target/www
+
```

Comment:
What's going on here? What is the artifact used for?

Reply:
I was using it to manually verify the website contents in this PR (to make sure it was built correctly). Otherwise, there isn't a way in a PR to verify what the website will look like (since it isn't deployed from other branches).

I am happy to either:

* Remove this
* Add a comment to say you can manually download the artifact to look at the website

What would you prefer?

Reply:
That’s clever. Sure just comment it. I assume artifacts are ephemeral?

Reply:
Yep, they are stored for 90 days by default it [looks like](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#about-workflow-artifacts)?

> By default, GitHub stores build logs and artifacts for 90 days, and this retention period can be customized. For more information, see "[Usage limits, billing, and administration](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#artifact-and-log-retention-policy)." The retention period for a pull request restarts each time someone pushes a new commit to the pull request.

Reply:
Added a comment which also links to this upstream issue for PR github pages deploys: https://github.com/orgs/community/discussions/7730